pkg/ebpf/events_definitions.go

```diff
@@ -6139,10 +6138,7 @@ var EventsDefinitions = map[int32]EventDefinition{
 		ID32Bit: sys32undefined,
 		Name:    "socket_dup",
 		Probes:  []probe{},
```

Comment:
Shouldn't the probes from the `DupEvent`s be flattened here the same as magic_write?

Reply:
No. These are tail calls, not probes.
We currently explicitly install the kprobes in tracee.go.
In a future PR, we should handle tail-calls dependencies of events.